scripts/zones/Newton_Movalpolos/mobs/Bugbear_Matman.lua

```diff
@@ -6,6 +6,17 @@ require("scripts/globals/hunts")
 -----------------------------------
 local entity = {}
 
+entity.onMobSpawn = function(mob)
+    mob:addMod(xi.mod.REGAIN, 50)
+end
+
+entity.onMobWeaponSkillPrepare = function(mob,target)
+    -- Below 30% Bugbear Matman heavily prefers Heavy Whisk
+	if mob:getHPP() <= 30 and math.random() > 0.4 then
```

Comment:
sneaky tab slipped in here. most editors have editorconfig support if you want conversion to be automated. Our project has a file that tells editorconfig enhances editor (and even git itself) what formatting to use.